NewForce-Cohort-6/tabloidfullstack-mashed-potatoes
# 22EditComment

# Description
[Trello Ticket #22 Edit Comment (https://trello.com/c/Y5Ddhq1x)]

Added feature to edit a comment on a post. The comment can be edited by clicking the edit button at the bottom of the comment. This will take you to a page with a form allowing the user to enter new text. This page will have a save button to confirm the edit to the comment and a link to cancel the edit by returning you to the comment.

# Testing Instructions
1. `git add` and `git commit` whatever you're working on on your branch
2. `git checkout main`
3. `git fetch --a`
4. `git checkout 22EditComment`
5. If you don't see the most recent version of my files, `git pull origin 22EditComments'
6. If you haven't created the Tabloid database, please do so.
Use this data to fill out tables and columns: [SQL Tables](https://github.com/NewForce-Cohort-6/tabloidfullstack-mashed-potatoes/blob/main/SQL/01_Tabloid_Create_DB.sql)
Use this to seed the database : [SQL SEED DATA](https://github.com/NewForce-Cohort-6/tabloidfullstack-mashed-potatoes/blob/main/SQL/02_Tabloid_Seed_Data.sql)
7. Click the execute with debugger solid green button to launch the app
8. Log into the website using “leo@gmail.com” if greeted with a login page
9. From the home page, click the “Post or My Post” option in the navbar
10. A list of posts will show on the page.
11. Choose a post to view. This will take you to the post details page with options to view comments, return to the post list or add a comment at the bottom of the page.
12. Click the view comment link. This will take you to a list of comments on the post. 
13. The bottom of each comment will have a edit button. Click the edit button which will take you to a page with a form allowing the user to enter new text.
14. First click the back to comments link to return you to the comment list.
15. Click the edit button again to return to the edit form page. Enter new text in the boxes and click save. This will edit the comment and return you to the updated comment list.



# Checklist:
